### ISSUE TITLE
Reference another NET 5 project result in error

### ISSUE THREAD

**Issue by mbendtsen** [PragmaticFlow/NBomber#306]
Runner: xUnit
Target framework: NET 5

I'm getting an error as soon as my unit test project references another class library, either targeting netstardard2.1 or net5. If I remove the reference everything is wokring fine. See details below.

The exception occurs on `OperatingSystem.GetObjectData` which according to [this](https://docs.microsoft.com/en-us/dotnet/core/compatibility/unsupported-apis) always throws `PlatformNotSupportedException` on NET 5.

**Exception**
```
  Message: 
    System.AggregateException : One or more errors occurred. (Operation is not supported on this platform.)
    ---- System.PlatformNotSupportedException : Operation is not supported on this platform.
  Stack Trace: 
    Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
    Task`1.GetResultCore(Boolean waitCompletionNotification)
    Task`1.get_Result()
    NBomberRunner.run(NBomberContext context)
    NBomberRunner.run(NBomberContext context)
    NBomberRunner.Run(NBomberContext context)
    PerformanceTest.SendingPackageToPersistance() line 91
    ----- Inner Stack Trace -----
    OperatingSystem.GetObjectData(SerializationInfo info, StreamingContext context)
    JsonSerializerInternalWriter.SerializeISerializable(JsonWriter writer, ISerializable value, JsonISerializableContract contract, JsonProperty member, JsonContainerContract collectionContract, JsonProperty containerProperty)
    JsonSerializerInternalWriter.SerializeObject(JsonWriter writer, Object value, JsonObjectContract contract, JsonProperty member, JsonContainerContract collectionContract, JsonProperty containerProperty)
    JsonSerializerInternalWriter.SerializeObject(JsonWriter writer, Object value, JsonObjectContract contract, JsonProperty member, JsonContainerContract collectionContract, JsonProperty containerProperty)
    JsonSerializerInternalWriter.Serialize(JsonWriter jsonWriter, Object value, Type objectType)
    JsonSerializer.SerializeInternal(JsonWriter jsonWriter, Object value, Type objectType)
    JsonSerializer.Serialize(JsonWriter jsonWriter, Object value, Type objectType)
    JsonConvert.SerializeObjectInternal(Object value, Type type, JsonSerializer jsonSerializer)
    HtmlReport.print(IGlobalDependency dep, TestInfo testInfo, NodeStats stats, FSharpList`1 timeLineStats)
    Report.build(IGlobalDependency dep, TestInfo testInfo, NodeStats nodeStats, FSharpList`1 timeLineStats)
    NBomberRunner.binder@1-9(TestInfo testInfo, NBomberContext context, IGlobalDependency dep, TestHost _arg3, NodeStats _arg4)
    runSession@41-4.Invoke(FSharpResult`2 r)
    RuntimeHelpers.DispatchTailCalls(IntPtr callersRetAddrSlot, IntPtr callTarget, IntPtr retVal)
    bindTaskConfigureFalse@131.Invoke(Unit unitVar0)
    TaskBuilder.tryFinally[a](FSharpFunc`2 step, FSharpFunc`2 fin)
    tryFinally@207-1.Invoke(Unit unitVar0)
    StepStateMachine`1.nextAwaitable()
    --- End of stack trace from previous location ---
    bindTaskConfigureFalse@131.Invoke(Unit unitVar0)
    StepStateMachine`1.nextAwaitable()
```

**Comment by mbendtsen** [comment]
Further digging shows that this issue occurs when referencing <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.0" />

**Comment by AntyaDev** [comment]
is it still reproducible with NBomber v2.0.0 beta ?
https://www.nuget.org/packages/NBomber/2.0.0-beta

**Comment by SimonSchulte** [comment]
Hey @AntyaDev , 
I stumbled across this issue when searching for the same not-supported exception, with a different stactrace though.

Upon investigation i can tell that in my case an unused reference to 
`<PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.7" />`
was causing this issue. After removing the reference, i got a clean shutdown of nbomber1.1.0.

The [2.0.0-rc3](https://www.nuget.org/packages/NBomber/2.0.0-beta) seems to solve this issue!

**Comment by AntyaDev** [comment]
@SimonSchulte To be honest I didn't have time to look at this issue. But sounds good if 2.0.0 fixing it.
btw: I already started updating docs for 2.0.0 https://nbomber.com/docs/overview/

**Comment by SimonSchulte** [comment]
@AntyaDev - i was not suggesting a need to fix this in 1.1.0. I only wante to give you some information, so that you might close this issue! :) 

After all, it is fixed! 
I will look into your docs

**Comment by AntyaDev** [comment]
@SimonSchulte  thanks:)
